capstone/feed-fetching/test_get_vt_feed.py

```diff
@@ -0,0 +1,48 @@
+import bz2
+import io
+import os
+import pytest
+import requests
+from datetime import datetime
+from unittest.mock import patch, Mock
+
+
+class MockResponse:
+    def __init__(self, status_code=200, text='', content=b''):
+        self.status_code = status_code
+        self.text = text
+        self.content = content
+
+
+@patch('os.environ.get', return_value='test-key')
+@patch('requests.get', return_value=MockResponse(status_code=200, content=b'vt-feed'))
+@patch('bz2.open', return_value=io.BytesIO(b'{ "attributes": { "attr1": "val1", "attr2": "val2" } }'))
+def test_get_feed(bz2_open, requests_get, os_environ):
```

Comment:
if you're using unittest, create a test class and add tests as methods from that class:
```python
class TestFeedFetching(unittest.TestCase):

    def test_get_feed(self,...):

    def test_prepare_doc(self,...):
```